test/compatibility/suite/test_flcs_deprecate.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python
+#
+# Public Domain 2014-present MongoDB, Inc.
+# Public Domain 2008-2014 WiredTiger, Inc.
+#
+# This is free and unencumbered software released into the public domain.
+#
+# Anyone is free to copy, modify, publish, use, compile, sell, or
+# distribute this software, either in source code form or as a compiled
+# binary, for any purpose, commercial or non-commercial, and by any
+# means.
+#
+# In jurisdictions that recognize copyright laws, the author or authors
+# of this software dedicate any and all copyright interest in the
+# software to the public domain. We make this dedication for the benefit
+# of the public at large and to the detriment of our heirs and
+# successors. We intend this dedication to be an overt act of
+# relinquishment in perpetuity of all present and future rights to this
+# software under copyright law.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+# IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+
+import compatibility_test, compatibility_version, wiredtiger, errno
+
+
+class test_flcs_deprecate(compatibility_test.CompatibilityTestCase):
+    '''
+    Test FLCS deprecation handling during database upgrade.
+    '''
+
+    build_config = {'standalone': 'true'}
+    conn_config = ''
+    create_config = 'key_format=r,value_format=8t'
+    uri = 'table:test_flcs_deprecate'
+    nrows = 100
+
+    def test_flcs_deprecate(self):
+
+        flcs_deprecated_version = compatibility_version.WTVersion(str("mongodb-8.3"))
+
+        # Test FLCS table creation fails on FLCS deprecated version
+        if self.older_branch > flcs_deprecated_version:
```

Comment:
As discussed, you will meed to implement the `__ge__` operator.
```suggestion
        if self.older_branch >= flcs_deprecated_version:
```

Reply:
added in [a614415](https://github.com/wiredtiger/wiredtiger/pull/12818/commits/a6144156bcb78f761a7df46ef3ff5c53c8fee5e8)